pkg/storage/mongo/mongo.go

```diff
@@ -56,7 +56,7 @@ func (m *ModuleStore) connect() error {
 
 func (m *ModuleStore) initDatabase() error {
 	// TODO: database and collection as env vars, or params to New()? together with user/mongo
-	m.d = "athens"
+	m.d = "admin"
```

Comment:
What do you think about fixing up the `TODO` above instead of changing this? I know that this would break a lot of deployments that already use mongo storage (including https://athens.azurefd.net)

Reply:
:+1:  Sure, I couldn't figure out how to talk to the emulator and I tried things out. It wasn't meant to stay here like this.